atomic_reactor/plugins/pre_inject_yum_repo.py

```diff
@@ -9,11 +9,38 @@
 Pre build plugin which injects custom yum repository in dockerfile.
 """
 import os
+from io import StringIO
+from urllib.parse import urlparse
 from atomic_reactor.constants import YUM_REPOS_DIR, RELATIVE_REPOS_PATH, INSPECT_CONFIG
 from atomic_reactor.plugin import PreBuildPlugin
+from atomic_reactor.plugins.pre_reactor_config import get_odcs
 from atomic_reactor.util import df_parser
 from atomic_reactor.utils.yum import YumRepo
 
+# The location of ca-bundle used during image build
+OSBS_BUILD_DIR = '/opt/osbs-build'
+BUILD_TIME_CA_BUNDLE = f'{OSBS_BUILD_DIR}/tls-ca-bundle.pem'
+BUILDER_CA_BUNDLE = '/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem'
+
+
+def inject_ca_bundle(dockerfile, base_from_scratch):
+    """Inject CA bundle into Dockerfile for accessing ODCS repos via HTTPS
+
+    :param dockerfile: the Dockerfile to inject the CA bundle.
+    :type dockerfile: DockerfileParser
+    :param bool base_from_scratch: whether the last stage is based on scratch
+        "FROM scrach", if it is a multistage build.
+    """
+    dockerfile.add_lines(
+        f'RUN mkdir -p {OSBS_BUILD_DIR}',
```

Comment:
when using /tmp, we don't have to care about existence of directory, neither in removing directory